pulp_rpm/app/advisory.py

```diff
@@ -142,6 +143,24 @@ def resolve_advisory_conflict(previous_advisory, added_advisory):
 
     """
 
+    def _datetime_heuristics(in_str):
+        # issue- and update-dates are supposed to be datetimes.
+        # Sometimes, they're empty. Sometimes, they're timestamps(?!?)
```

Comment:
"timestamp" is a little ambiguous, if we're talking about Unix/epoch time then let's say that.

For what it's worth I do think createrepo_c special-case allows this: https://github.com/rpm-software-management/createrepo_c/blob/master/src/python/updaterecord-py.c#L402

Reply:
It's exactly the Python/POSIX definition of timestamp, yes.

I expect it's **not** an "intended" usecase, but more a "well we never had a standard and it's too late to fix now" usecase. Use of timestamps is inconsistent - SUSE, for example, doesn't use it, except for a couple of advisories. Usage of timestamp has caused problems w/ advisories in the past, because of the lack of timezone info and the impacts of releasing an advisory during the DST timechange window.

However, it is what it is, we need to handle it, and this fix does so.